### DISCUSSION TITLE
observation-beta-shared-state-syncups Branch Missing Dependancies

### DISCUSSION THREAD

**Discussion by BakrMohamedRBM** [pointfreeco/swift-composable-architecture#2823] [Beta]
Dear Team,

I am working with observation-beta-shared-state-syncups branch to use Shared functionality but I encounter an issue with dependencies

Err: could not find a branch named **‘diffable-object’** in https://github.com/pointfreeco/swift-custom-dump

is there another safe branch to work with or a migration plan to main in nearest future

thanks for your assistance and great effort really appreciated <3

**Comment by acosmicflamingo** [comment]
They haven't made an official beta announcement so it's safe to assume that any branch like that is experimental and broken. Hopefully tonight or tomorrow they make an announcement and then we have an official branch we can start using.

Edit: forgot that they were releasing the livestream this week so maybe next week it'll be released, but we'll see 🙂

**Comment by mbrandonw** [comment]
Yes, these tools are not officially released as a beta feature yet, and so you really should not be depending on that branch.
